mindsdb/api/executor/utilities/sql.py

```diff
@@ -147,6 +150,12 @@ def adapt_query(node, is_table, **kwargs):
             else:
                 if user_functions is not None:
                     user_functions.check_function(node)
+        # If a variable is used in the target list, add quotes around the value.
+        if isinstance(node, Variable):
+            # Variables in the targets will cause an error in DuckDB due to the '@@' or '@' prefix.
```

Comment:
what query didn't work before this update?

Reply:
This is the query that was broken, Andrey:
`select @@session.auto_increment_increment, @@character_set_client;`